src/govuk/components/pagination/pagination.yaml

```diff
@@ -0,0 +1,198 @@
+params:
+- name: classes
+  type: string
+  required: false
+  description: The classes that you want to add to the pagination.
+- name: previous
+  type: object
+  required: false
+  description: A link to the previous page, if there is one.
+  params:
+  - name: text
+    type: string
+    required: false
+    description: The previous page link text. Defaults to 'Previous page', where 'page' is visually hidden.
+  - name: href
+    type: string
+    required: true
+    description: The previous page URL.
+- name: next
+  type: object
+  required: false
+  description: A link to the next page, if there is one.
+  params:
+  - name: text
+    type: string
+    required: false
+    description: The next page link text. Defaults to 'Next page', where 'page' is visually hidden.
+  - name: href
+    type: string
+    required: true
+    description: The next page URL.
+- name: items
+  type: array
+  required: true
+  description: Array of link objects.
+  params:
+  - name: type
+    type: string
+    required: false
+    description: Set to 'elipses' to indicate where a list has been truncated.
```

Comment:
"Ellipses" is spelt with two `l`s, which I think needs to be corrected here, in the CSS and in the Nunjucks template.

Reply:
Good spot! Fixed